resourceService/resourceService.env

```diff
@@ -0,0 +1,18 @@
+DATASOURCE_DATABASE_URL=jdbc:mysql://localhost:3307/resourcedb?createDatabaseIfNotExist=true&autoReconnect=true&useSSL=false
```

Comment:
Do you think it is a correct JDBC URL? I think it should not be pointed to "localhost"